### PR TITLE
Created KickAssFingers presence

### PR DESCRIPTION
**Pull request by eBeckrs**
Created the first version of kickassfingers.com presence.

Screenshots showing a side-by-side comparison of my profile and the website:
screenshot

### CODE REVIEW

File: websites/K/KickAssFingers/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "author": {
+    "name": "Beckrs",
+    "id": "777552923177386025"
+  },
+  "service": "KickAssFingers",
+  "altnames": ["KAF"],
```
Comment (Bas950) [inline websites/K/KickAssFingers/dist/metadata.json:8]:
```suggestion
```

File: websites/K/KickAssFingers/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "author": {
+    "name": "Beckrs",
+    "id": "777552923177386025"
+  },
+  "service": "KickAssFingers",
+  "altnames": ["KAF"],
+  "description": {
+    "en": "A typing website with a modern user interface and a customisable user experience. Test your typing speed on a different typing tests, earn points, win medals and climb the rankings.",
+    "it": "Un sito di test di battitura con una moderna interfaccia e una esperienza utente personalizzabile. Testa tua velocità di battitura in diversi tipi di test, guadagna punti, vinci medaglie e scala le classifiche."
```
Comment (Bas950) [inline websites/K/KickAssFingers/dist/metadata.json:11]:
```suggestion
    "en": "Test your typing speed on a different typing tests, earn points, win medals and climb the rankings.",
    "it": "Needs updating..."
```

File: websites/K/KickAssFingers/presence.ts [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  }),
+  time = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+
+   const presenceData: PresenceData = {
+      largeImageKey: "logo"
+   };
+   
+   presenceData.startTimestamp = time;
```
Comment (Bas950) [inline websites/K/KickAssFingers/presence.ts:12]:
```suggestion
   const presenceData: PresenceData = {
      largeImageKey: "logo",
      startTimestamp: time
   };
```

File: websites/K/KickAssFingers/presence.ts [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+      }
+   }
+
+   if(!document.querySelector("#cntr-boxOverlay").classList.contains("displayNone")) {
+      presenceData.details = vars.checking_stats;
+      delete presenceData.state;
+   }
+ 
+   if(presenceData.details === null) {
```
Comment (Bas950) [inline websites/K/KickAssFingers/presence.ts:68]:
```suggestion
   if(!presenceData.details) {
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
run prettier with our settings.

File: websites/K/KickAssFingers/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "author": {
+    "name": "Beckrs",
+    "id": "777552923177386025"
+  },
+  "service": "KickassFingers",
```
Comment (Bas950) [inline websites/K/KickAssFingers/dist/metadata.json:7]: Service and folder name must be 100% the same (caps included)

File: websites/K/KickAssFingers/presence.ts [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp: time
+  },
+
+   data = document.querySelector("#premIdVars"),
+    vars = JSON.parse(data.getAttribute("vars")),
+    live = JSON.parse(data.getAttribute("liveScores"));
```
Comment (Bas950) [inline websites/K/KickAssFingers/presence.ts:14]:
```suggestion
    data = document.querySelector("#premIdVars"),
    vars = JSON.parse(data.getAttribute("vars")),
    live = JSON.parse(data.getAttribute("liveScores"));
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**